### PR TITLE
CIAM-3049 After new upgrade of keycloak-pipeline nodes, ssso containe…

### PR DESCRIPTION
**Pull request by drichtarik**
…rs are incompatible with cekit 4

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,5 @@
 # RHSSO-1862: Ignore the "fetch-artifacts-pnc.yaml" file, generated by CPaaS pipeline run
```
Comment (iankko) [inline .gitignore]: @drichtarik Could you remove that comment line altogether?

File: .gitignore [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
-/fetch-artifacts-pnc.yaml
+# /fetch-artifacts-pnc.yaml
```
Comment (iankko) [inline .gitignore]: Maning, not just comment it out, but completely remove it

**Comment by drichtarik** [comment]
Done

**Review by iankko — APPROVED** [review]
@drichtarik Thanks, Denis. Verified all of referenced:

- Artifact names,
- PNC build IDs, and
- PNC Artifact IDs are correct

Extra points for scheduling dedicated/fresh PNC build today to get these :+1:

**Pull request merged**